### DISCUSSION TITLE
Failed to generate ConfigBuilder for extension when execute cache:clear

### DISCUSSION THREAD

**Discussion by fd6130** [symfony/symfony#42893] [Q&A] [answered]
Hi, when i execute `php bin/console cache:clear` i saw some errors on the terminal:

```
15:34:58 WARNING   [app] Failed to generate ConfigBuilder for extension SymfonyBundles\JsonRequestBundle\DependencyInjection\JsonRequestExtension. ["exception" => Symfony\Component\DependencyInjection\Exception\EnvNotFoundException^ { …},"extensionClass" => "SymfonyBundles\JsonRequestBundle\DependencyInjection\JsonRequestExtension"]
15:34:58 WARNING   [app] Failed to generate ConfigBuilder for extension SamJ\FractalBundle\DependencyInjection\SamJFractalExtension. ["exception" => Symfony\Component\DependencyInjection\Exception\EnvNotFoundException^ { …},"extensionClass" => "SamJ\FractalBundle\DependencyInjection\SamJFractalExtension"]

...
```

What will happen if some bundle doesn't have ConfigBuilder and we wish to deploy to production server? Will it affect our project in production? Thanks.

**Comment by ahundiak** [comment, marked as answer]
It should not make a difference unless you are actually using the generated config builder classes in your configuration.  Which, if you were, would then generate it's own set of class not found errors.  So as long as the cache appears to be built cleanly then you should be okay.

Both bundles have an Extension class but no Configuration class.  The Configuration class is used to generate the builders.  I'm speculating but it could just be an unnecessary warning that should be suppressed by the builder code.

**Reply by fd6130** [reply]
Thx for the answer!
